include/picongpu/plugins/saxs/jupyter/validate.ipynb

```diff
@@ -0,0 +1,4975 @@
+{
```

Comment:
@ejcjason can you please add this file as python modules in `lib/python/picongpu/plugins/data` (data parsing/reading) and `lib/python/picongpu/plugins/plot_mpl` (plotting)?